beetsplug/parentwork.py

```diff
@@ -0,0 +1,165 @@
+        super(ParentWorkPlugin, self).__init__()
+        self.import_stages = [self.imported]
+
+    def commands(self):
+        cmd = ui.Subcommand('parentwork',
+                            help=u'fetches parent works, composers \
+                                and performers')
+        cmd.func = self.command
+        return [cmd]
+
+    def command(self, lib, opts, args):
+        self.find_work(lib.items(ui.decargs(args)))
+
+    def imported(self, session, task):
+        self.find_work(task.items)
+
+    def find_work(self, items):
+
+        
+
+        for item in items:
+            performer            = []
+            performer_sort       = []
+            work                 = []
+            work_disambig        = []
+            parent_work          = []
+            parent_work_disambig = []
+            parent_composer      = []
+            parent_composer_sort = []
+            work_ids             = set()
+            composer_ids         = set()
+            
+            item.read()
+            recording_id = item['mb_trackid']
+            performer_types = ['performer', 'instrument', 'vocal',
+                'conductor', 'performing orchestra', 'chorus master', 
+                    'concertmaster']
+            found = True
+            if item.get('parent_work')!=None:
+                continue
+            try:
+                rec_rels = musicbrainzngs.get_recording_by_id(
+                    recording_id, includes=['work-rels', 'artist-rels'])
+                if 'artist-relation-list' in rec_rels['recording']:
+                    for dudes in rec_rels['recording'][
+                    'artist-relation-list']:
+                        if dudes['type'] in performer_types:
+                            performer.append(dudes['artist']['name'])
+                            performer_sort.append(dudes['artist']
+                                ['sort-name'])
+                if 'work-relation-list' in rec_rels['recording']:
+                    for work_relation in rec_rels['recording'][
+                            'work-relation-list']:
+                        if work_relation['type'] != 'performance':
+                            continue
+                        work_id = work_relation['work']['id']
+                        work.append(work_relation['work']['title'])
+                        if 'disambiguation' in work_relation['work']:
+                            work_disambig.append(work_relation['work']
+                                ['disambiguation'])
+                        work_info = find_parentwork(work_id)
+                        get_info(work_info,parent_composer,
+                        parent_composer_sort,parent_work,parent_work_disambig,
+                        work_ids,composer_ids)
+
+            except musicbrainzngs.musicbrainz.WebServiceError: 
+                print('Work unreachable')
+                print('recording id: ')
+                print(recording_id)
```

Comment:
The `_log` object comes from Python's standard `logging` module. It might be worth taking a look at other plugins' source code to see how it's used.